bin/catalog-api-approval-minion

```diff
@@ -24,3 +24,10 @@ args = parse_args
 
 catalog_api_minion_approval = Catalog::Api::Minion::Approval.new(args[:queue_host], args[:queue_port])
 catalog_api_minion_approval.run
+
+require "catalog-api-client"
+CatalogApiClient.configure do |config|
+  config.scheme = ENV["CATALOG_SCHEME"] || "http"
+  config.host   = "#{ENV["CATALOG_HOST"]}:#{ENV["CATALOG_PORT"]}"
+  config.logger = Catalog::Api::Minion.logger
+end
```

Comment:
This will need to move up above the `.run` call, at the minimum. I moved mine above `args = parse_args`, but it doesn't really matter.